userdocs/src/usage/eks-managed-nodes.md

```diff
@@ -119,85 +115,80 @@ version. You cannot roll back a nodegroup to an earlier Kubernetes version.
 the latest AMI release for Kubernetes 1.14 using:
 
-```console
+```
 eksctl upgrade nodegroup --name=managed-ng-1 --cluster=managed-cluster --kubernetes-version=1.14
 ```
 
-### Nodegroup Health issues
-EKS Managed Nodegroups automatically checks the configuration of your nodegroup and nodes for health issues and reports
-them through the EKS API and console.
+## Nodegroup health issues
+
+EKS Managed Nodegroups automatically checks the configuration of your nodegroup and nodes for health issues and reports them through the EKS API and console.
 To view health issues for a nodegroup:
 
-```console
+```
 eksctl utils nodegroup-health --name=managed-ng-1 --cluster=managed-cluster
 ```
 
-### Managing Labels
+## Managing labels
+
 EKS Managed Nodegroups supports attaching labels that are applied to the Kubernetes nodes in the nodegroup. This is
 specified via the `labels` field in eksctl during cluster or nodegroup creation.
 
 To set new labels or updating existing labels on a nodegroup:
 
-```console
+```
 eksctl set labels --cluster managed-cluster --nodegroup managed-ng-1 --labels kubernetes.io/managed-by=eks,kubernetes.io/role=worker
 ```
 
 
 To unset or remove labels from a nodegroup:
 
-```console
+```
 eksctl unset labels --cluster managed-cluster --nodegroup managed-ng-1 --labels kubernetes.io/managed-by,kubernetes.io/role
 ```
 
 To view all labels set on a nodegroup:
 
-```console
+```
 eksctl get labels --cluster managed-cluster --nodegroup managed-ng-1
 ```
 
-### Scaling Managed Nodegroups
+## Scaling managed modegroups
 `eksctl scale nodegroup` also supports managed nodegroups. The syntax for scaling a managed or unmanaged nodegroup is
 the same.
 
-```console
+```
 eksctl scale nodegroup --name=managed-ng-1 --cluster=managed-cluster --nodes=4
 ```
 
+## Feature parity with unmanaged nodegroups
 
-### Feature parity with unmanaged nodegroups
 EKS Managed Nodegroups are managed by AWS EKS and do not offer the same level of configuration as unmanaged nodegroups.
+
 The unsupported options are noted below.
 
 - No support for private networking (`nodeGroups[*].privateNetworking`).
 - Tags (`managedNodeGroups[*].tags`) in managed nodegroups apply to the EKS Nodegroup resource and do not propagate to
 the provisioned Autoscaling Group like in unmanaged nodegroups.
-- `iam.instanceProfileARN` is not supported for managed nodegroups.
+- `iam.instanceProfileARN` and `iam.instanceRoleARN` are not supported for managed nodegroups.
 - The `amiFamily` field supports only `AmazonLinux2`
 - `instancesDistribution` field is not supported
 - `volumeSize` is the only field supported for configuring volumes
 - Control over the node bootstrapping process and customization of the kubelet are not supported. This includes the
-following fields: `classicLoadBalancerNames`, `maxPodsPerNode`, `taints`, `targetGroupARNs`, `preBootstrapCommands`, `overrideBootstrapCommand`,
```

Comment:
classicLoadBalancerNames should be there. Please check that this file is up to date with the latest changes in master